ddmrp/models/stock_buffer.py

```diff
@@ -371,6 +371,15 @@ def action_view_mrp_productions(self):
         result["domain"] = [("id", "in", mrp_production_ids.ids)]
         return result
 
+    product_type = fields.Selection(related="product_id.type", readonly=True)
+    used_in_bom_count = fields.Integer(related="product_id.used_in_bom_count")
+
+    def action_used_in_bom(self):
```

Comment:
For methods that doesn't update records data and returns an action, I use `button_` prefix (like the `button_validate` in `stock` module, used from the UI, while the validation itself is handled by `action_done`).